src/SFA.DAS.Roatp.CourseManagement.Web/Views/Standards/ViewStandards.cshtml

```diff
@@ -16,45 +16,34 @@
-        <p class="govuk-body">Check your standards are correct and add any missing information. Add any standards that you deliver that are not on the list.</p>
-        <p class="govuk-body">These details are what employers see on Find apprenticeship training. It is important you come back and make updates when they change.</p>
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl" id="header-standards">Manage your standards</h1>
+        <p class="govuk-body">Select <strong>Add a standard</strong> to add a new course, or select the name of a standard to change its training options or contact details. You can also delete a standard.</p>
+        <p class="govuk-body">Any changes you make will update Find apprenticeship training.</p>
+        <p class="govuk-body">
+            <a href="@Model.AddAStandardLink" class="govuk-button">
+                Add a standard
+            </a>
+        </p>
         <table class="govuk-table" aria-labelledby="header-standards">
             <thead class="govuk-table__head">
-                <tr class="govuk-table__row">
-                    <th class="govuk-table__header das-table-cell-width-60" scope="col"> Standard</th>
-                    <th class="govuk-table__header das-table-cell-width-30" scope="col"> <span class="govuk-visually-hidden">Approval status</span> </th>
-                    <th class="govuk-table__header govuk-table__header--numeric das-table-cell-width-10" scope="col"> Version</th>
-                </tr>
+            <tr class="govuk-table__row">
+                <th class="govuk-table__header govuk-!-font-size-24" scope="col">Your standards</th>
+            </tr>
             </thead>
             <tbody class="govuk-table__body">
-                @foreach (var standard in Model.Standards.OrderBy(x=>x.CourseDisplayName))
-                {
-                    <tr class="govuk-table__row">
-                        <td class="govuk-table__cell">
-                            <a class="govuk-link" href="@standard.StandardUrl">
-                                @standard.CourseDisplayName
-                            </a>
-                        </td>
-                        <td class="govuk-table__cell govuk-table__cell--numeric">
-                            @if (standard.IsApprovalPending)
-                            {
-                                <a class="govuk-link" href="@standard.ConfirmRegulatedStandardUrl">Regulator's approval needed<span class="govuk-visually-hidden"> for @standard.CourseDisplayName</span></a>
-                            }
-                        </td>
-                        <td class="govuk-table__cell govuk-table__cell--numeric">
-                            @standard.Version
-                        </td>
-                    </tr>
-                }
+            @foreach (var standard in Model.Standards.OrderBy(x => x.CourseDisplayName))
```

Comment:
Feels a little odd we're ordering in the view rather than the model

Reply:
this isn't a change implemented in this story

Reply:
ok this is now sorted in code